### DISCUSSION TITLE
Limit the length of tiddler titles when clipping tweets

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#104] [Q&A]
I frequently use tiddlyclip to clip from tweets, but such clipped tweet tiddler's have lengthy title like shown below. 
 
<img width="1703" alt="image" src="https://user-images.githubusercontent.com/67494083/190924733-11d6fdc3-cac3-4717-9ac1-525f6f68b85e.png">

How to limit the length of tiddler titles without losing access to this tiddler for the subsequent tweet clippings. 

I tried using regexp method mentioned [here](https://github.com/buggyj/tiddlyclip-plugin/discussions/47), but the subsequent clips wont be saved in first tiddler or I will have to use [this method](https://github.com/buggyj/tiddlyclip-plugin/discussions/72). Both are time consuming. Another way is to use `userstring` for tiddler titles, but this also has to be repeated for every clips.  Currently what I do is - rename the tiddler after the clipping is over, but any future clippings to this tiddler will have to use [this method](https://github.com/buggyj/tiddlyclip-plugin/discussions/72)
Is there any other way to limit the length of tiddler titles without losing access to this tiddler for the subsequent tweet clippings.

**Comment by buggyj** [comment]
Exactly how would you like to limit the title? number of characters?

**Reply by arunnbabu81** [reply]
I cant think of any way other than limiting the number of characters.
Usually how I rename these tiddler is like this - `Tweet/authorname/topic`. This I could achieve by using the regexp method you taught before. But if I use that method, the subsequent clips aren't added to the first clipped tiddler.

So limiting the number of characters must be the only option rite?

**Comment by buggyj** [comment]
what happens to the subsequent clips?

**Reply by arunnbabu81** [reply]
> what happens to the subsequent clips?

Subsequent tweets should get the same title.

**Comment by buggyj** [comment]
I think that subsequent clips will have the same title when reduced with the regex, and therefore should be appended to the already existing clip tiddler.

**Reply by arunnbabu81** [reply]
You are right. Subsequent titles are appended to the already existing tiddler when regexp is used. I have to find a way to use it efficiently. Will come back after testing for some more days. Thanks for your help.
